src/components/Card/Card.module.scss

```diff
@@ -1,40 +1,49 @@
 @import "../../assets/variables.scss";
 
+$image-size: 47px;
+
 .card {
-    width: 100%;
-    border: solid 2px $grey003;
-    border-radius: 4px;
-    padding: $M-spacing;
-    display: flex;
-    align-items: center;
-    margin-bottom: $M-spacing;
+  width: 100%;
+  border: solid 2px $grey003;
+  border-radius: 4px;
+  padding: $M-spacing;
+  display: flex;
+  align-items: center;
+  margin-bottom: $M-spacing;
 
-    &__text {
-        display: flex;
-        flex-direction: column;
+  &__text {
+    display: flex;
+    flex-direction: column;
 
-        & > p {
-            margin-top: $S-spacing;
-            font-weight: $regular;
-        }
+    & > p {
+      margin-top: $S-spacing;
+      font-weight: $regular;
     }
+  }
 
-    &__image-container {
-        height: 47px;
-        width: 47px;
-        margin-right: $M-spacing;
-    }
+  &__image-container {
+    height: $image-size;
+    width: $image-size;
+    margin-right: $M-spacing;
+    display: flex;
 
-    &__right-content {
-        margin-left: auto;
+    img {
+      max-width: $image-size;
+      object-fit: contain;
```

Comment:
Co powiesz na to, żeby zamiast `contain` użyć `cover`? Przy dużej dysproporcji wysokości i długości obrazka może to różnie wyglądać 😅

Reply:
a można, zdecydowanie, jak najbardziej - update w https://github.com/akai-org/EaterUI/pull/72/commits/0509bfbb7cc15f2601caa5f91ddf71ce80d0550f